commit 1993e23d59e37ee7befbc64bf1535640a16354bc
Author: Olivier Houchard <ohouchard@haproxy.com>
Date:   Tue Oct 16 18:39:38 2018 +0200

    MINOR: cfgparse: Write 130 as 128 as 0x82 and 0x80.
    
    Write 130 and 128 as 8x82 and 0x80, to avoid warnings about casting from
    int to size. "check_req" should probably be unsigned, but it's hard to do so.
    
    (cherry picked from commit 3332090a2d3e9e84bac67af79fb03be111359429)
    Signed-off-by: Willy Tarreau <w@1wt.eu>

diff --git a/src/cfgparse.c b/src/cfgparse.c
index 7414b60d..87a4d803 100644
--- a/src/cfgparse.c
+++ b/src/cfgparse.c
@@ -5082,7 +5082,7 @@ stats_error_parsing:
 									((unsigned char) (packetlen >> 16) & 0xff));
 
 								curproxy->check_req[3] = 1;
-								curproxy->check_req[5] = 130;
+								curproxy->check_req[5] = 0x82; // 130
 								curproxy->check_req[11] = 1;
 								curproxy->check_req[12] = 33;
 								memcpy(&curproxy->check_req[36], mysqluser, userlen);
@@ -5108,7 +5108,7 @@ stats_error_parsing:
 								((unsigned char) (packetlen >> 16) & 0xff));
 
 							curproxy->check_req[3] = 1;
-							curproxy->check_req[5] = 128;
+							curproxy->check_req[5] = 0x80;
 							curproxy->check_req[8] = 1;
 							memcpy(&curproxy->check_req[9], mysqluser, userlen);
 							curproxy->check_req[9 + userlen + 1 + 1]     = 1;
